### PR TITLE
Feature/dsdev 1909 binding failures

### PR DESCRIPTION
**Pull request by ssaucedo**

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/vault/VaultApp.scala [hunk 1]
```diff
@@ -1,34 +1,16 @@
...
-
   def main(args: Array[String]) {
     log.info("Vault instance starting.")
-    start()
-  }
-
-  def start(): Unit = {
-    // start a new HTTP server on configuration port with our service actor as the handler
-    IO(Http) ? Http.Bind(service, VaultConfig.HttpConfig.interface, VaultConfig.HttpConfig.port)
+    ServerInitializer.startWebServiceActors(Props[VaultServiceActor],VaultConfig.HttpConfig.interface,VaultConfig.HttpConfig.port,VaultConfig.HttpConfig.timeout,system)
```
Comment (kshakir) [inline src/main/scala/org/broadinstitute/dsde/vault/VaultApp.scala]: Pleaseaddspacestothisline. :trollface: 

`⌥⌘L` in IntelilJ should do it in one keystroke.

File: src/main/scala/org/broadinstitute/dsde/vault/VaultConfig.scala [hunk 2]
```diff
@@ -9,6 +9,8 @@ object VaultConfig {
     private val httpConfig = config.getConfig("http")
     lazy val interface = httpConfig.getString("interface")
     lazy val port = httpConfig.getInt("port")
+    lazy val timeout = httpConfig.getLong("timeout")
```
Comment (kshakir) [inline src/main/scala/org/broadinstitute/dsde/vault/VaultConfig.scala]: Put the units in the name, aka `timeoutSeconds`.

**Comment by kshakir** [comment]
Minor updates. Update, bump `vault-common` in build.sbt with the (soon to be released) new version, rebase onto latest develop, squash commits, and good to go!

**Comment by jmthibault79** [comment]
Please rebase onto latest origin/develop and squash to two commits: ServerInitializer and API versioning.

**Comment by jmthibault79** [comment]
One comment: changing these tests and services would mean dropping support for the unversioned endpoints in the API.  I don't believe that's desirable at the moment.

**Comment by ssaucedo** [comment]
I took as example how it was handled for UbamCollectionServiceSpec , thought there was a special reason. Im  fixing it...

**Comment by jmthibault79** [comment]
Ah, I see.  `UbamCollectionService` is different from these because it is a new service.  We have never published that API to any other groups, so we have the ability to require versioning.

These older services are used versionless by other software, so we need to maintain support for them currently.  Sometime in the future, all services will be versioned.

**Comment by jmthibault79** [comment]
Looks good otherwise.  Could you restore `ApiPrefix` and squash again down to the two commits?

**Comment by jmthibault79** [comment]
Thanks! :+1:

**Pull request merged**